### ISSUE TITLE
Reintroduce the usage of PyPI for package management

### ISSUE THREAD

**Issue by 4lm** [OpenEnergyPlatform/oemetadata#53]
Hi @christian-rli and @jh-RLI, I was on PyPI and saw that this library is not kept up-to-date ([there](https://pypi.org/project/oep-metadata/) `v1.0.1`, here `v1.4.1-fixed`). Therefore my questions:

- Who are the maintainers of this project right now?
- On PyPI @Ludee, @MGlauer, and I have access rights for releasing. Shall I give further access rights to you or other people? Who should this be?
- There is a minor bug in the pip install command described in the readme. I introduced this two years ago by replacing all `metadata` strings with `oemetadata` :/
- I can write a quick PR to fix this issue and improve the PyPI release description in the readme so that a yet to be defined maintainer can create new releases from thereon. Shall I do this and write a new PR?

BTW: I think it's not a good idea to intertwine the metadata versioning with this codebase's versioning. These are IMO two separate things. If you decouple them, you don't need helpers like `-fixed` in `v1.4.1-fixed`, because following this logic the next release might be something like  `v1.4.1-fixed-changed` or `v1.4.1-fixed-fixed`. I think you should have versioning for the metadata standard and versioning for the codebase itself. Because my idea initially was that this codebase is a monorepo in which (i) the metadata standard is defined and (ii) the different concrete implementations in whatever language you need are also included (right now only Python, but could be any other language as well). Shall I write an issue for this?

**Comment by jh-RLI** [comment]
Sure, I will definitely do that once the release is ready. Currently we don't have access to the Pypi repository for https://pypi.org/project/oep-metadata/.

**Comment by jh-RLI** [comment]
The package index is now updated to v1.5.0.
See https://pypi.org/project/oep-metadata/
